### ISSUE TITLE
How to use vendors css class without getting scoped to component? (ex. Bootstrap)

### ISSUE THREAD

**Issue by Alex-Ferreli** [nairinarinyan/react-scoped-styles#1]
Hi, i'm trying to configure this plugin in my `CRA`. It's awesome, so easy to implement.

But i didn't figure out how to use global CSS class from vendor CSS like `bootstrap` without get this class scoped too.

Maybe the plugin should scope only classes that are present in CSS file and not all classes by default, but i'm not sure that script-loader can read style-loader result.

There is an easy way to do that?

Thanks in advance!

**Comment by nairinarinyan** [comment]
You are absolutely right, script loader and style loader are completely separate functions.

It is easy to detect if the imported stylesheet comes from `node_modules` and ignore in style loader. And on scripts side maybe read the imported style file if possible, and rule out the classes which aren't found in the styles.

I plan to add this feature but usage and implementation details are not clear atm.

**Comment by salahmedamin** [comment]
Hey @nairinarinyan , has this issue been fixed yet ?
